docs/reference/namespaces.md

```diff
@@ -0,0 +1,174 @@
+
+This type of namespace is used when multiple parties need to share on- and off-chain data and agree upon the ordering and authenticity of that data.
+
+### Gateway Namespaces
+
+Nothing in this namespace will be shared automatically, and no assumptions are made about whether other parties connected through this namespace are also using Hyperledger FireFly. Plugins for data exchange and shared storage are not supported. If any identities or definitions are created in this namespace, they will be stored in the local database, but will not be shared implicitly outside the node.
+
+This type of namespace is mainly used when interacting directly with a blockchain, without assuming that the interaction needs to conform to FireFly's multi-party system model.
+
+## Configuration
+
+FireFly nodes can be configured with one or many namespaces of different modes. This means that a single FireFly node can be used to interact with multiple distinct blockchains, multiple distinct token economies, and multiple business networks.
+
+Below is an example plugin and namespace configuration containing both a multi-party and gateway namespace:
+
+```
+plugins:
+  database:
+  - name: database0
+    type: sqlite3
+    sqlite3:
+      migrations:
+        auto: true
+      url: /etc/firefly/db?_busy_timeout=5000
+  blockchain:
+  - name: blockchain0
+    type: ethereum
+    ethereum:
+      ethconnect:
+        url: http://ethconnect_0:8080
+        topic: "0"
+  - name: blockchain1
+    type: ethereum
+    ethereum:
+      ethconnect:
+        url: http://ethconnect_01:8080
+        topic: "0"
+  dataexchange:
+  - name: dataexchange0
+    type: ffdx
+    ffdx:
+      url: http://dataexchange_0:3000
+  sharedstorage:
+  - name: sharedstorage0
+    type: ipfs
+    ipfs:
+      api:
+        url: http://ipfs_0:5001
+      gateway:
+        url: http://ipfs_0:8080
+  tokens:
+  - name: erc20_erc721
+    broadcastName: erc20_erc721
+    type: fftokens
+    fftokens:
+      url: http://tokens_0_0:3000
+namespaces:
+  default: alpha
+  predefined:
+  - name: alpha
+    description: Default predefined namespace
+    defaultKey: 0x123456
+    plugins: [database0, blockchain0, dataexchange0, sharedstorage0, erc20_erc721]
+    multiparty:
+      networkNamespace: alpha
+      enabled: true
+      org:
+        name: org0
+        description: org0
+        key: 0x123456
+      node:
+        name: node0
+        description: node0
+      contract:
+        - location:
+            address: 0x4ae50189462b0e5d52285f59929d037f790771a6
+          firstEvent: 0
+        - location:
+            address: 0x3c1bef20a7858f5c2f78bda60796758d7cafff27
+          firstEvent: 5000
+  - name: omega
+    defaultkey: 0x48a54f9964d7ceede2d6a8b451bf7ad300c7b09f
+    description: Gateway namespace
+    plugins: [database0, blockchain1, erc20_erc721]
+```
+
+The `namespaces.predefined` object contains the follow sub-keys:
```

Comment:
I was going to suggest cross-linking **to** this section, from the config reference doc... but in the TOC I couldn't find the config reference doc 🤔 
... could you check why I can't find it?

Reply:
![image](https://user-images.githubusercontent.com/6660217/188205681-0e9dba00-29e5-4593-a3c7-43d34b798eaf.png)

Reply:
Not sure why it was missing, but fixed it now 🤷🏻‍♂️